src/frontend/efiling-frontend/src/components/page/home/Home.stories.js

```diff
@@ -21,8 +22,9 @@ const confirmationPopup = getTestData();
 const page = { header, confirmationPopup };
 
 const submissionId = "abc123";
-const mock = new MockAdapter(axios);
+const temp = "temp";
 const apiRequest = `/submission/${submissionId}`;
+const apiRequestFiling = `/submission/${submissionId}/filing-package`;
```

Comment:
mabay `getFiliingPackagePath` or something similar, the one before is also just `apiRequest` it might be confusing